shallow-server/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM debian:buster
+FROM debian:stable
```

```suggestion
FROM debian:bullseye
```

I would advise, even if updating to the latest version, to use a specific one (instead of `stable`). `stable` is a moving tag which will  eventually change to point to the next version (when `bullseye` becomes `oldstable`) and will break this build without our input.

This also applies to the `server` image, which currently uses `stable` as well.

---

Not sure if there's any policy regarding this @nickvergessen

---

I have no stocks in shallow-server